### DISCUSSION TITLE
Student status

### DISCUSSION THREAD

**Discussion by BelX-mk49** [community/community#41663] [GitHub Education]
I'm a student Virginia Highland Community College, but the automatic system didn't approve my evidence. The support team didn't help me. What do I need to do?

**Comment by jung-youjin** [comment]
Have you tried validating your status as a student with your school email address domain? 📚 🎒

**Reply by BelX-mk49** [reply]
> Have you tried validating your status as a student with your school email address domain? 📚 🎒
Yes, but for github's, it isn't enough. I have mail on edu domain

**Reply by jung-youjin** [reply]
You should try validating your student status with you edu domain email. That worked for me.

**Reply by BelX-mk49** [reply]
> You should try validating your student status with you edu domain email. That worked for me.

My account is linked with edu email, but i don't know what I need to do next

**Reply by jung-youjin** [reply]
You may apply for [Student Developer Pack](https://education.github.com/pack) which is one of the benefits of a student can get from GitHub. Also you can look out for more [programs](https://education.github.com/students) for students. All these would be accessible if you have successfully validated your status as a student! 😄

**Reply by BelX-mk49** [reply]
![Screenshot_20221214_103651_Samsung Internet](https://user-images.githubusercontent.com/25936829/207534213-48bc122e-bd9c-4965-bcd9-2940aefdf2db.jpg)
always get that error

**Reply by samus-aran** [reply]
Hi @BelX-mk49 👋 

Let's see how we can assist. 

The rejection reasons asks that if you are a remote learner (based on the campus location) that your evidence needs demonstrates this. Do you have a description of the course in the evidence stating it's remote or virtual?

**Reply by BelX-mk49** [reply]
> Hi @BelX-mk49 👋 
> 
> Let's see how we can assist. 
> 
> The rejection reasons asks that if you are a remote learner (based on the campus location) that your evidence needs demonstrates this. Do you have a description of the course in the evidence stating it's remote or virtual?

I don't learn remotely, and I don't know what type of evidence needs to confirm my student status

**Reply by samus-aran** [reply]
Hi @BelX-mk49 

Are you attending a campus of this school when applying? You state that your school is Virginia Highland Community College, are you close to this school?

**Reply by BelX-mk49** [reply]
yep, I'm in
